frontend/src/Utils.js

```diff
@@ -31,7 +31,7 @@ export const fetchSonsgByName = async (songName, setSongs) => {
 export const fetchTopSongs = async (setTopSongs) => {
   try {
     const response = await axios.get("https://spring-music-player-3hyj.vercel.app/search?song=top songs");
-    const jsonData = response.data.slice(0,5);
+    const jsonData = response.data.slice(0,20);
```

Comment:
make a new function for this otherwise it will have conflicts main page top songs block.
and backend I need to change 20songs for this.

Reply:
Keeping the function lke this we can use the `slice` function on `topSongs` array and get the number of songs we want. So, we won't need to add separate functions for different quantity of topSongs?

Reply:
@The-Phoenics fine whenever I'll get time i'll increase the response limit from backend.